pkgs/development/python-modules/requests-gssapi/default.nix

```
@@ -0,0 +1,27 @@
+{ fetchPypi
+, lib
+, python3Packages
+}:
+python3Packages.buildPythonPackage rec {
+  pname = "requests-gssapi";
+  version = "1.2.3";
+  src = fetchPypi {
```

Upstream says Python > [3.8](https://github.com/pythongssapi/requests-gssapi/blob/main/pyproject.toml#L11).

```suggestion
  version = "1.2.3";
  pyproject = true;

  disabled = pythonOlder "3.8";

  src = fetchPypi {
```